akka-stream/src/main/scala/akka/stream/javadsl/Sink.scala

```diff
@@ -379,6 +385,12 @@ final class Sink[In, Mat](delegate: scaladsl.Sink[In, Mat]) extends Graph[SinkSh
    */
   def asScala: scaladsl.Sink[In, Mat] = delegate
 
+  /**
+   * Connect this `Sink` to a `Source` and run it.
+   */
```

Comment:
maybe add something like this to all the `run*(ClassicActorSystemProvider)` methods in javadsl:
```
Note that the `ActorSystem` can be used as the `systemProvider` parameter.
```